### PR TITLE
rough reimplementation of the AntiVanish Module

### PR DESCRIPTION
**Pull request by Razzy52**
The old AntiVanish had absolutely no working code, I think it'd be better to have something even if it's not perfect.

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt [hunk 1]
```diff
@@ -1,48 +1,40 @@
...
-//                if (it.packet.entries.size != mc.player?.networkHandler?.playerList?.size) {
-//                    notification("AntiVanish", message("staffDetected"), NotificationEvent.Severity.INFO)
-//                } else {
-//                    notification("AntiVanish", message("allClear"), NotificationEvent.Severity.INFO)
-//                }
-//            }
-//
-//            else -> {}
-//        }
+        val packet = event.packet as PlayerListS2CPacket
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt]: Use Kotlin smart-case
https://kotlinlang.org/docs/typecasts.html#smart-casts

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt [hunk 2]
```diff
@@ -1,48 +1,40 @@
...
- * (at your option) any later version.
- *
- * LiquidBounce is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
- */
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt]: leave this

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt [hunk 3]
```diff
@@ -1,48 +1,40 @@
...
 
+import net.ccbluex.liquidbounce.event.events.NotificationEvent
 import net.ccbluex.liquidbounce.event.events.PacketEvent
 import net.ccbluex.liquidbounce.event.handler
 import net.ccbluex.liquidbounce.features.module.Category
 import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.utils.client.notification
 import net.minecraft.network.packet.s2c.play.PlayerListS2CPacket
 
+    /**
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/ModuleAntiVanish.kt]: weird formatting

**Comment by Razzy52** [comment]
@1zun4 should be okay to merge unless there's something else im missing?

**Comment by 1zun4** [comment]
It makes no sense. It always says "Staff detected" when someone leaves the server? You are checking the size of the tablist against the size of the tablist.

**Pull request closed without merging**